pulser/_seq_drawer.py

```diff
@@ -199,6 +202,44 @@ def phase_str(phi):
         a.set_ylabel(r'$\Omega$ (rad/µs)', fontsize=14, labelpad=10)
         b.set_ylabel(r'$\delta$ (rad/µs)', fontsize=14)
 
+        if draw_phase_area:
+            top = False  # Variable to track position of box, top or center.
+            draw_phase = any(
+                seq_.type.phase != 0 for seq_ in seq._schedule[ch]
+                if isinstance(seq_.type, Pulse)
+            )
+            for pulse_num, seq_ in enumerate(seq._schedule[ch]):
+                # Select only `Pulse` objects
+                if isinstance(seq_.type, Pulse):
+                    if sampling_rate:
+                        area_val = np.sum(cs_amp(
+                            np.arange(seq_.ti, seq_.tf)
```

Comment:
nit: Please check if you can avoid the line break (I can't tell from here)

Reply:
Actually, I took yet another look at the code, and I think you need to adjust the indices to `cs_amp` according to `time_scale`, like this:
```suggestion
                            np.arange(seq_.ti, seq_.tf) / time_scale
```
Can you please confirm this by doing an extra plot where the total duration of the sequence goes over 10 µs (this will change the `time_scale`). Just add a large delay to the end of your current test sequence.

Reply:
### Plots

(Below are two examples)

a)
1. **Sequence**

![f_seq1](https://user-images.githubusercontent.com/68844397/119676186-4bec2280-be5b-11eb-9bfa-c38c3c99ff2c.png)

2. **Simulation**

![f_sim1](https://user-images.githubusercontent.com/68844397/119676244-560e2100-be5b-11eb-9df6-db458e09c7f2.png)

b)
1. **Sequence**

![f_seq2](https://user-images.githubusercontent.com/68844397/119677969-cb2e2600-be5c-11eb-91af-fe917169ca23.png)

2. **Simulation**

![f_sim2](https://user-images.githubusercontent.com/68844397/119678006-d5502480-be5c-11eb-94e4-3aec0b550bb4.png)

---
#### Example of the long duration *simulation* plot

![long_dur_sim](https://user-images.githubusercontent.com/68844397/119679198-ca49c400-be5d-11eb-96ea-e9f504227f53.png)

The time scale on the x-axis does change from `ns` to `µs`

Reply:
> nit: Please check if you can avoid the line break (I can't tell from here)

Does the current commit's line break look good? It needs a bit of adjustment for `flake8` to pass :)

> Alright, this all looks correct. This is with the time_scale adjustment, right?

Yes, the plots are with the `time_scale` adjustment. Thanks for that too!